src/components/banner/template.test.js

```diff
@@ -9,9 +9,14 @@ const examples = getExamples('banner')
 describe('Internal Header', () => {
   describe('by default', () => {
     it('passes accessibility tests', async () => {
-      const $ = render('new-tab-link', examples.default)
+      const $ = render('internal-header', examples.default)
+
+      const results = await axe($.html(), {
+        rules: {
+          'region': { enabled: false }
```

Comment:
I feel that we should be looking to provide more life-like examples instead of turning off rules - maybe this can be done at a later time so it doesn't bloat the security update.